docs/stdout_publisher.md

```diff
@@ -0,0 +1,129 @@
+# Using the stdout publisher
+
+Enabling this with `DEYE_FEATURE_STDOUT_PUBLISHER` will dump all collected
+metrics in json on stdout once every cycle (every `DEYE_DATA_READ_INTERVAL`
+seconds).
+This can be useful in combination with other tools to process and push the
+metrics into different systems other than mqtt.
+
+The json object has the following structure:
+```json
+{
+  "serial": "1234567890",
+  "address": "192.168.0.1",
+  "port": 8899,
+  "data": [
+    {
+      "up": 1,
+    }, {
+      "temp": 31,
+      "name": "Radiator temperature",
+      "unit": "°C",
+      "groups": "micro",
+      "sensor": "SingleRegisterSensor",
+      "source": "radiator",
+      "timestamp": 1686219130
+    }, {
+      "energy": 0.2,
+      "name": "PV1 Total",
+      "unit": "kWh",
+      "groups": "micro",
+      "sensor": "DoubleRegisterSensor",
+      "source": "dc/pv1/total",
+      "timestamp": 1686219130
+    }, {
+      ...
+    }
+  ]
+}
+```
+
+## Example usage with [telegraf]
+
+To collect the inverter metric with telegraf and then process or push them
+forward you can run `deye-inverter-mqtt` with the following configuration:
+- disable the mqtt publisher
+- enable the stdout publisher
+- ensure all logs go to stderr instead of stdout
+
+This can be achieved with a telegraf config file like that:
+```
+[[outputs.file]]
+  files = ["stdout"]
+  data_format = "influx"
+
+[[inputs.execd]]
+  command = [
+    "docker",
+    "run",
+    "-e", "DEYE_LOGGER_IP_ADDRESS=192.168.0.254",
+    "-e", "DEYE_LOGGER_PORT=8899",
+    "-e", "DEYE_LOGGER_SERIAL_NUMBER=123412341234",
+    "-e", "LOG_LEVEL=INFO",
+    "-e", "LOG_STREAM=STDERR",
+    "-e", "DEYE_FEATURE_MQTT_PUBLISHER=false",
+    "-e", "DEYE_FEATURE_STDOUT_PUBLISHER=true",
+    "-e", "DEYE_FEATURE_SET_TIME=false",
+    "-e", "DEYE_METRIC_GROUPS=micro",
+    "ghcr.io/kbialek/deye-inverter-mqtt",
```

Comment:
@kbialek I switched the telegraf example to run `deye-inverter-mqtt` as a container. This would only work if you plan to publish the new version of the container image with the `latest` tag, which I think you do, right?

Reply:
Yes, the most recent release is tagged as `latest`